test/index.js

```diff
@@ -273,31 +273,61 @@ describe('logger', () => {
         expect(this.output, 'to equal', fixtures[title]);
     });
 
-    it('should log an unhandled error', async function () {
-        const title = this.test.fullTitle();
-        const createLogger = this.createLogger(title);
+    describe('when an error is thrown', () => {
+        it('should log the error', async function () {
+            const title = this.test.fullTitle();
+            const createLogger = this.createLogger(title);
 
-        const logger = createLogger();
+            const logger = createLogger();
 
-        const context = {
-            method: 'GET',
-            originalUrl: '/'
-        };
+            const context = {
+                method: 'GET',
+                originalUrl: '/'
+            };
 
-        const next = () => {
-            const error = new Error();
-            error.stack = 'Error\n    at stack';
+            const next = () => {
+                const error = new Error();
+                error.stack = 'Error\n    at stack';
 
-            throw error;
-        };
+                throw error;
+            };
 
-        try {
-            await logger(context, next);
-        } catch (error) {
-            // Not interested in this one
-        }
+            try {
+                await logger(context, next);
+            } catch (error) {
+                // Not interested in this one
+            }
 
-        expect(this.output, 'to equal', fixtures[title]);
+            expect(this.output, 'to equal', fixtures[title]);
+        });
+
+        it('should log the error status if avaiable', async function () {
+            const title = this.test.fullTitle();
+            const createLogger = this.createLogger(title);
+
+            const logger = createLogger();
+
+            const context = {
+                method: 'GET',
+                originalUrl: '/'
+            };
+
+            const next = () => {
+                const error = new Error('GatewayTimeout');
```

Comment:
>  We need to make sure that it produces the same output for all environments

Yeah I figured that's why the stack was overwritten. I'm not sure either why the new one has the curly brackets..

It's not a custom error, I'm using this to throw the error:
```js
app.use(async context => {
    context.throw(503, 'Yo');
});
```
Then koa prepends the error message with 'ServiceUnavailableError: '. Also note that with the actual error (in [the comment above](https://github.com/PabloSichert/concurrency-logger/pull/5#discussion_r86428407)), the error doesn't have curly brackets. Making it all the more puzzing :)